### PR TITLE
Worker Activity report fixed douple counitng

### PR DESCRIPTION
**Pull request by lwyszomi**
@czue 
http://manage.dimagi.com/default.asp?175773#1108151

### CODE REVIEW

File: corehq/apps/reports/analytics/esaccessors.py [hunk 1]
```diff
@@ -64,6 +64,25 @@ def _get_case_case_counts_by_owner(domain, datespan, case_types, is_total=False)
     return case_query.run().aggregations.owner_id.counts_by_bucket()
 
 
+def get_active_case_count(domain, datespan, case_types, is_total=False):
```
Comment (benrudolph) [inline corehq/apps/reports/analytics/esaccessors.py]: any chance we can get a test for this? we have a pretty extensive suite going here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/tests/test_esaccessors.py. reports are one of those areas in our code base that doesn't have a lot of coverage so at least making sure the database access is tested is good!

**Comment by czue** [comment]
:+1: agree with @benrudolph on adding a test

**Comment by lwyszomi** [comment]
@czue @benrudolph I added tests.

**Comment by benrudolph** [comment]
thanks

**Pull request merged**